### ISSUE TITLE
Streaming mode without buffering multiple lines?

### ISSUE THREAD

**Issue by mark19142218** [chmln/sd#302]
I noticed that all my output gets buffered until the command completes. Example

```shell
bash -c "echo STDOUT && echo STDERR 1>&2; sleep 5" | sd -f=e 'a' 'b'
```

Error is shown immediately, but output only after 5 seconds.

This happens with `0.7.6` and with `v1.0.0 - 2023-11-07`.

I don't see an option in `--help` to avoid this.

Possibly related to #96 ?

**Comment by mark19142218** [comment]
Might be duplicate of #286 actually
